website/preprints/tasks.py

```diff
@@ -172,12 +172,14 @@ def format_preprint(preprint, share_type, old_subjects=None):
     return [node.serialize() for node in visited]
 
 
-def get_and_set_preprint_identifiers(preprint):
-    ezid_response = request_identifiers_from_ezid(preprint)
-    if ezid_response is None:
+@celery_app.task(ignore_results=True)
+def get_and_set_preprint_identifiers(preprint_id):
```

Comment:
A guid is expected as the argument here but, a [few lines up in this file](https://github.com/CenterForOpenScience/osf.io/pull/8344/files#diff-ff363b707e0ef078be1004dd0193cbc6R29), a `PreprintService` object is passed. Are you sure this is working?